### PR TITLE
feature/input_copy_chrExt

### PR DESCRIPTION
**Pull request by Wojtekkas**

### CODE REVIEW

File: extension/mtu.js [hunk 1]
```diff
@@ -1,14 +1,37 @@
-document.body.style.backgroundColor = "silver";
-
 chrome.runtime.onMessage.addListener(
     function(request, sender, sendResponse) {
         console.log(sender.tab ?
             "from a content script:" + sender.tab.url :
             "from the extension");
-        var inputValue = request.inputValue
+        var nameValue = request.nameValue
```
Comment (MKwestorowski) [inline extension/mtu.js:6]: What request doing here? Is this method from chrome function? Please explain

File: extension/popup.html [hunk 2]
```diff
@@ -31,9 +31,39 @@
...
+        </div>
+      </div>
+    <div style="display: flex">
+      <div style="padding: 15px">
+      <p>Podaj Przebieg:</p>
+      <input id="kilometers" type="number"/>
+      </div>
+      <div style="padding: 15px">
+      <p>Data prawa jazdy:</p>
+      <input id="licenseDate" placeholder="__-__-____" type="date"/>
```
Comment (MKwestorowski) [inline extension/popup.html:61]: There we need validation, please try to do it before merge to develop. Same thing happens in 61 line. Good Luck :) !

File: extension/mtu.js [hunk 3]
```diff
@@ -1,14 +1,37 @@
...
+        else if (50001 <= kilometersValue && kilometersValue < 100001 )
+        {kilometersActive = 75000}
+        else if (100001 <= kilometersValue && kilometersValue  < 150001)
+        {kilometersActive = 125000}
+        else if (150001 <= kilometersValue && kilometersValue  < 200001)
+        {kilometersActive = 175000}
+        else
+        {kilometersActive = 225000}
+
+        if (licenseValue)
```
Comment (MKwestorowski) [inline extension/mtu.js]: I cant get this if, is this thing doing something here? I think this isn't, may I'm dumb. Please explain.

**Review by MKwestorowski — APPROVED** [review]
Im fine with this, anyway i want to know why we need this vcs.xml. I think we should make some css file to style popup. Look at line comments, if u can explain some thing today, at daily. This should be great. Have good time ! :)

File: extension/mtu.js (hunk 3 above)
Comment (Wojtekkas) [reply extension/mtu.js]: fixed

File: extension/popup.html (hunk 2 above)
Comment (Wojtekkas) [reply extension/popup.html:61]: Validation is on our todo list. I will do it later.

File: extension/mtu.js (hunk 1 above)
Comment (Wojtekkas) [reply extension/mtu.js:6]: Request is for communicating between content script and extension. You need to use request if u want to handle the message that had been sent to you from one of this two above mentioned.

**Review by falbor45 — CHANGES_REQUESTED** [review]
Other than removal of .idea/vcs.xml file and changing id names into a proper camelCase notation (@popup.html #L(37/41/47/51) & @popup.js #L2), I have no objections, yet I'd rather have it done before upcoming merge so that we don't leave ourselves with excessive amount of redundant files on that very branch.

File: extension/mtu.js (hunk 1 above)
Comment (RafalPaluch) [reply extension/mtu.js:6]: You mean of couse by extension background scripts

File: extension/manifest.json [hunk 4]
```diff
@@ -10,7 +10,7 @@
   },
```
Comment (RafalPaluch) [inline extension/manifest.json:10]: Can we treat content script and background script as the same file. I mean mtu.js. I though content script deal with other things than background script and these 2 files should coummunicate with each other etc. Please explain.( I do not mark this line, please open the file)

File: extension/popup.js [hunk 5]
```diff
@@ -1,28 +1,41 @@
...
+var nameInput = document.getElementById('nameCalc')
+var kilometersInput = document.getElementById('kilometers')
+var peselInput = document.getElementById('peselCalc')
+var surnameInput = document.getElementById('surnameCalc')
+var zipcodeInput = document.getElementById('zipCodeCalc')
+var licenseInput = document.getElementById('licenseDate')
 function updateInput(event) {
-  findExtInput.setAttribute("value", event.target.value)
+    nameInput.setAttribute("value", event.target.value)
 }
```
Comment (RafalPaluch) [inline extension/popup.js:10]: Is that function necessery?

File: extension/popup.js [hunk 6]
```diff
@@ -1,28 +1,41 @@
...
 
 document.addEventListener('DOMContentLoaded',function() {
-    findExtInput.onchange = updateInput;
+    nameInput.onchange = updateInput;
 },false);
```
Comment (RafalPaluch) [inline extension/popup.js:14]: Is that addeventListener necessery? Like above function it only gets Nameimput variable, so i guess whole  script should work wothout it.  Please explain.

File: extension/popup.js (hunk 5 above)
Comment (MKwestorowski) [reply extension/popup.js:10]: This function is necessery, absolute. This lines, setting value of current input.

File: extension/popup.js (hunk 6 above)
Comment (MKwestorowski) [reply extension/popup.js:14]: U must set listeners on document to look for findExtInput and also nameInput. Then we have to do something if someone is typing in current input. Thats why its `currentInput.onchange`
If someone change content of input areas this line call out updateInput function. That last one function is explained above this post.

File: extension/popup.js (hunk 5 above)
Comment (RafalPaluch) [reply extension/popup.js:10]: Not current. Only one input. That's why I asked. Try to delete it and find the differences.

File: extension/popup.js (hunk 6 above)
Comment (RafalPaluch) [reply extension/popup.js:14]: Line with FindeExtInput is actually deleted. I know even by name what the function is doing :). Please go above this post. The question is valid for this line also. Thanks anyway for answer:)))

File: extension/popup.js (hunk 5 above)
Comment (MKwestorowski) [reply extension/popup.js:10]: Take look at next pull request by Artur. This functions were fixed by adding another tags with ids. Have a look at deleted line. There was a random input name. Now changed at unique that present name.

Comment (RafalPaluch) [reply extension/popup.js:10]: ...

**Pull request merged**